Core/DISET/private/BaseClient.py

```diff
@@ -205,7 +205,7 @@ def __findServiceURL( self ):
       
     if len( urls ) == len( self.__bannedUrls ):
       self.__bannedUrls = []  # retry all urls
-      self.__retryDelay = 5 / len ( urls )  # we run only one service! In that case we increase the retry delay.
+      self.__retryDelay = 5 / len ( urls ) if len( urls ) > 1 else 1  # we run only one service! In that case we increase the retry delay.
```

Comment:
no, What I want is to have maximum 2.5 second delay. This is the case when we run 2 service. But when we run more service I want the numbers to be decreased. For example: 5/3=1.66, 5/4=1.25
So, If we have more service, it make no sense to wait to much, because we have to retry very fast all the services.
I want to keep a constant number (which is 1) when 1 service running.